js/WebView.android.js

```diff
@@ -152,8 +152,8 @@ class WebView extends React.Component<WebViewSharedProps, State> {
         javaScriptEnabled={this.props.javaScriptEnabled}
         thirdPartyCookiesEnabled={this.props.thirdPartyCookiesEnabled}
         domStorageEnabled={this.props.domStorageEnabled}
-        messagingEnabled={typeof this.props.onMessage === 'function'}
         onMessage={this.onMessage}
+        messagingEnabled={typeof this.props.onMessage === 'function'}
```

Comment:
avoid theses kind of changes as they will mostly just create merge conflicts

Reply:
I kept these from @KoenLav's changes and resolved the conflicts when merging from master. Let me know if you want me to change it back.